packages/cli/src/cli.ts

```diff
@@ -19,12 +19,14 @@ import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 import type { CliCommonArgs } from "./CliCommonArgs.js";
 import auth from "./commands/auth/index.js";
-import site from "./commands/site/index.js";
+import siteHandler from "./commands/site/index.js";
 import typescript from "./commands/typescript/index.js";
 import { ExitProcessError } from "./ExitProcessError.js";
+import { logConfigFileMiddleware } from "./util/configLoader.js";
```

Comment:
Let's put this together with existing middleware

```suggestion
import { logConfigFileMiddleware } from "./yargs/configLoader.js";
```